src/components/Operations/SortRemoveMotion/sort-remove--motion.module.scss

```diff
@@ -0,0 +1,135 @@
+
+
+@keyframes sort-remove-line1 {
+  0% {
+    transform: translateX(0px);
+  }
+  10% {
+    transform: translateX(6px);
+  }
+  20% {
+    transform: translateX(0px);
+  }
+  100% {
+    transform: translateX(0px);
+  }
+}
+
+@keyframes sort-remove-line2 {
+  0% {
+    transform: translateX(0px);
+  }
+  10% {
+    transform: translateX(-6px);
+  }
+  20% {
+    transform: translateX(0px);
+  }
+  100% {
+    transform: translateX(0px);
+  }
+}
+
+@keyframes sort-remove-line3 {
+  0% {
+    transform: translateX(0px);
+  }
+  10% {
+    transform: translateX(-2px);
+  }
+  20% {
+    transform: translateX(0px);
+  }
+  100% {
+    transform: translateX(0px);
+  }
+}
+
+
+@keyframes sort-remove-vertical {
+  0% {
+    transform: scaleY(1);
+  }
+  10% {
+    transform:  scaleY(.7);
+  }
+  20% {
+    transform: scaleY(1);
+  }
+  100% {
+    transform: scaleY(1);
+  }
+}
+
+@keyframes sort-remove-horizontal {
+  0% {
+    transform: scaleX(1);
+  }
+  10% {
+    transform:  scaleX(.7);
+  }
+  20% {
+    transform: scaleX(1);
+  }
+  100% {
+    transform: scaleX(1);
+  }
+}
+
+.SortRemoveVertical,
+.SortRemoveHorizontal {
+  will-change: transform;
+  rotate: 45deg;
```

Comment:
I think usually we use `transform: rotate()`, but looks like `rotate` has decent although fairly recent support.
https://developer.mozilla.org/en-US/docs/Web/CSS/rotate#browser_compatibility

Reply:
Good article on this topic: https://danielcwilson.com/blog/2020/02/motion-path-transforms/